main.go

```diff
@@ -134,6 +134,11 @@ func probeHandler(w http.ResponseWriter, r *http.Request, c *config.Config, logg
 	debugOutput := DebugOutput(&module, &sl.buffer, registry)
 	rh.Add(moduleName, target, debugOutput, success)
 
+	// log out debug ouput if probe failed
+	if !success {
+		level.Error(logger).Log("msg", debugOutput)
```

Comment:
I think this should be `Debug` level, rather than error.

Reply:
We already log this, see about 10 lines up.